src/libraries/System.Private.CoreLib/src/System/Text/Unicode/Utf16Utility.Validation.cs

```diff
@@ -15,15 +15,12 @@
 #pragma warning disable SA1121 // explicitly using type aliases instead of built-in types
 #if SYSTEM_PRIVATE_CORELIB
 #if TARGET_64BIT
-using nint = System.Int64;
-using nuint = System.UInt64;
+using nuint_t = System.UInt64;
```

Comment:
These aliases didn't go away?

Reply:
Ah, I see. It is used in `Vector<T>`....

Reply:
Do you think it's worthwhile extending `Vector<T>` to work with `nint`/`nuint` given we have a use-case here in the runtime (not necessarily immediately, but just something for the general backlog)?
I believe it would be fairly straightforward to handle in the importer since we could just treat it as `Int32` or `Int64` and wouldn't need to expand the tables...

Reply:
Yes, it sounds reasonable to me.

Reply:
Logged https://github.com/dotnet/runtime/issues/36160